Dockerfile.dev

```diff
@@ -0,0 +1,36 @@
+# Use Python 3.10 slim as the base image
+FROM python:3.10
+RUN apt-get update && apt-get install -y git-all
+
+# Upgrade pip
+RUN pip install --upgrade pip
+
+# Set the working directory to /app
+WORKDIR /app
+VOLUME psycop-common
+
+# Dev experience
+## Lazygit
+RUN LAZYGIT_VERSION=$(curl -s "https://api.github.com/repos/jesseduffield/lazygit/releases/latest" | grep -Po '"tag_name": "v\K[^"]*') && curl -Lo lazygit.tar.gz "https://github.com/jesseduffield/lazygit/releases/latest/download/lazygit_${LAZYGIT_VERSION}_Linux_x86_64.tar.gz" && tar xf lazygit.tar.gz lazygit && install lazygit /usr/local/bin && rm -rf lazygit lazygit.tar.gzRUN curl https://raw.githubusercontent.com/jesseduffield/lazydocker/master/scripts/install_update_linux.sh | bash
```

Comment:
what does this do?

Reply:
Install a command-line utility called lazygit.

https://github.com/jesseduffield/lazygit